ui/src/components/header/HeaderMenu.js

```diff
@@ -87,68 +92,103 @@ const HeaderMenuUserDiv = styled.div`
-    let icon = 'notification';
-    const router = useRouter();
-    useEffect(() => {
-        props.getUserPendingMembers();
-    });
-    if (props.pending) {
-        if (Object.keys(props.pending).length !== 0) {
-            icon = 'notification-solid';
-        }
+const ReviewNotificationSpan = styled.span`
+    position: absolute;
+    top: 10px;
+    right: 90px;
+    width: 15px;
+    height: 15px;
+    background-color: red;
+    border-radius: 50px;
+`;
+
+class HeaderMenu extends React.Component {
+    constructor(props) {
+        super(props);
+        this.api = API();
+        this.state = {
+            roleGroupReviewFeatureFlag: false,
+        };
     }
-    let menuItems =
-        props.headerDetails &&
-        props.headerDetails.headerLinks &&
-        props.headerDetails.headerLinks.map((headerLink, idx) => {
-            return (
-                <MenuItemDiv key={idx}>
-                    <StyledAnchorDiv
-                        data-testid='menu'
-                        onClick={() =>
-                            window.open(headerLink.url, headerLink.target)
-                        }
-                    >
-                        {headerLink.title}
-                    </StyledAnchorDiv>
-                </MenuItemDiv>
-            );
+
+    onClickFunction(route) {
+        this.props.router.push(route, route);
+    }
+
+    componentDidMount() {
+        this.props.getUserPendingMembers();
+        this.props.getReviewRoles();
+        this.props.getReviewGroups();
+        this.api.getPageFeatureFlag('roleGroupReview').then((data) => {
+            this.setState({
+                roleGroupReviewFeatureFlag: data['roleGroupReviewFeatureFlag'],
+            });
         });
-    return (
-        <HeaderMenuDiv data-testid='header-menu'>
-            <Icon
-                icon={icon}
-                isLink
-                onClick={() => router.push(PageUtils.workflowAdminPage())}
-                size={'25px'}
-                color={colors.white}
-            />
-
-            <HeaderMenuUserDiv>
-                <Menu
-                    placement='bottom-end'
-                    trigger={({ getTriggerProps, triggerRef }) => (
-                        <Icon
-                            icon={'help-circle'}
-                            {...getTriggerProps({ innerRef: triggerRef })}
-                            isLink
-                            size={'25px'}
-                            color={colors.white}
-                        />
-                    )}
-                    triggerOn='click'
-                >
-                    <MenuDiv>{menuItems}</MenuDiv>
-                </Menu>
-            </HeaderMenuUserDiv>
-            {props.headerDetails && props.headerDetails.userId && (
+    }
+
+    render() {
+        let props = this.props;
+        let reviewNotificationRedDot = '';
+        let icon = 'notification';
+        let clickIcon = this.onClickFunction.bind(
```

Comment:
future note: when you add the new tabs for role/group review, update this to go to the tab which has pending approval work.

Reply:
oh interesting good suggestion